clusters/gitlab-cluster/dashboard/dashboard.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kubernetes-dashboard
+
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
+metadata:
+  name: kubernetes-dashboard
+  namespace: flux-system
+spec:
+  interval: 1m
+  url: https://kubernetes.github.io/dashboard/
+
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dashboard
+  namespace: kubernetes-dashboard
+spec:
+  interval: 5m
+  chart:
+    spec:
+      chart: kubernetes-dashboard
+      version: "4.0.3"
```

Comment:
:thinking-taco:  Nothing to change in this PR, but how do we keep all these little versions up to date that are scattered about?   I don't have an answer.   Versioning: I love it cause it lets me understand the state of the world for a given commit.  Versioning: I hate it cause those versions need to be updated at some point and that feels like toil.

Reply:
I think that right now, we should KISS, but longer term, it would be easy to write a script to scrape all the helmreleases deployed in the cluster and do a "helm repo search <release>" and compare the versions.  That works for helm.  For other non-helm things, I have been trying to write update.sh scripts which we might be able to run to see if anything changes.

I would imagine the first step would be to just let us know "hey, X could be updated".  If we reaaaallly wanted to, we could then automatically update git with the latest versions, but I think that is dangerous, especially until we get good tests going.  This is a great example of automating shooting yourself in the foot with the safety engaged.  It'll probably work, but what if it broke, or somebody flipped it off while you weren't looking.

I don't really like the idea of updating things automatically, and neither does k8s and the fluxcd project either.  See their reasoning in #2 of the list of differences under https://docs.fluxcd.io/projects/helm-operator/en/stable/helmrelease-guide/introduction/#a-minimal-helmrelease .